index.js

```diff
@@ -1,7 +1,20 @@
 const readline = require('readline-sync');
+const calc = require('./calculator');
 
 console.log('Welcome to the calculator!');
 
-console.log('Please enter some input:');
-const response = readline.prompt();
-console.log('You entered: ' + response);
+console.log('Enter the operator:');
+const operator = readline.prompt();
+
+console.log(`How many numbers do you want to ${operator} ?`);
+const c = parseInt(readline.prompt());
+
+const calculator = calc.create_calculator(operator);
+
+for (var i = 0; i < c; i++) {
```

Comment:
You've used `let` to declare your variables in "calculator.js" and `var` here. It is good practice to be consistent with how you set your variables. Generally `let` is the preferred option, and there are very few cases that `var` would _need_ to be used instead.

Reply:
"very few cases that var would need to be used instead." 

None that I know of